lib/sprockets.rb

```diff
@@ -197,6 +199,17 @@ module Sprockets
   register_mime_type 'application/html+ruby', extensions: ['.html.erb', '.erb', '.rhtml'], charset: :html
   register_mime_type 'application/xml+ruby', extensions: ['.xml.erb', '.rxml']
 
+  require 'sprockets/exporters/file_exporter'
+  register_exporter '*/*', Exporters::FileExporter
+
+  # Gzip
+  require 'sprockets/exporters/zlib_exporter'
+  require 'sprockets/exporters/zopfli_exporter'
+  if defined?(::Zopfli)
```

Comment:
I believe we should use a config not autoload the thing if the constant is defined, otherwise we may have surprising behavior.